tests/repl_stats.test/runit

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Test replication stats
+#
+# Force replication delay on a replicant and then do a pushnext on the master
+# We should be seeing the replicant is behind the master in master's
+# comdb2_repl_stats table.
+
+bash -n "$0" | exit 1
+
+set -e
+
+dbnm=$1
+
+master=`cdb2sql --tabs ${CDB2_OPTIONS} $dbnm default 'select host from comdb2_cluster where is_master="Y"'`
+rep=`cdb2sql --tabs ${CDB2_OPTIONS} $dbnm default 'select host from comdb2_cluster where is_master="N" limit 1'`
+
```

Comment:
One quick note: the test would timeout if run against a standalone setup. I'd skip the test in that case.
`
if [[ -z "${rep}" ]]; then
    echo "skipping, it's a cluster test"
    exit 0;
fi
`